README.md

```diff
@@ -1,237 +1,81 @@
-wget https://github.com/containernetworking/cni/releases/download/v0.4.0/cni-amd64-v0.4.0.tbz2
-sudo tar xjf cni-amd64-v0.4.0.tbz2 -C /opt/cni/bin/
-```
+Start the VMs and save a snapshot for quicker iteration:
 
-### Copy Configuration Files
+```bash
+vagrant up
+vagrant snapshot save clean
 ```
-sudo cp assets/auth/kubeconfig /etc/kubernetes/
-sudo cp -a assets/manifests /etc/kubernetes/
-```
-### Start the Kubelet
-```
-sudo hyperkube kubelet --kubeconfig=/etc/kubernetes/kubeconfig \
-    --require-kubeconfig \
-    --cni-conf-dir=/etc/kubernetes/cni/net.d \
-    --network-plugin=cni \
-    --lock-file=/var/run/lock/kubelet.lock \
-    --exit-on-lock-contention \
-    --pod-manifest-path=/etc/kubernetes/manifests \
-    --allow-privileged \
-    --node-labels=master=true \
-    --minimum-container-ttl-duration=6m0s \
-    --cluster_dns=10.3.0.10 \
-    --cluster_domain=cluster.local \
-    --hostname-override=10.7.183.59
-```
-The TLS credentials generated by `bootkube render` in assets/tls/ are copied to a secret: assets/manifests/kube-apiserver-secret.yaml.
 
-### Start the Temporary API Server
-bootkube will serve as the temporary apiserver so the kubelet from above can start the real apiserver in a pod
-```
-sudo bootkube start --asset-dir=./assets  --experimental-self-hosted-etcd --etcd-server=http://127.0.0.1:12379
-```
-bootkube should exit itself after successfully bootstrapping the master components. It's only needed for the very first bootstrapping
+Spin up a cluster:
 
-### Check the Output
-```
-watch hyperkube kubectl get pods -o wide --all-namespaces
+```bash
+./test-install.sh
 ```
 
-## Join Nodes to the Cluster
-Copy the information where to find the apiserver and how to authenticate:
-```
-scp 10.7.183.59:assets/auth/kubeconfig .
-sudo mkdir -p /etc/kubernetes
-sudo mv kubeconfig /etc/kubernetes/
-```
-install cni binaries and download hyperkube
-```
-sudo mkdir -p /opt/cni/bin
-wget https://github.com/containernetworking/cni/releases/download/v0.4.0/cni-amd64-v0.4.0.tbz2
-sudo tar xjf cni-amd64-v0.4.0.tbz2 -C /opt/cni/bin/
-wget http://storage.googleapis.com/kubernetes-release/release/v1.5.3/bin/linux/amd64/hyperkube -O ./hyperkube
-sudo mv hyperkube /usr/bin/hyperkube
-sudo chmod 755 /usr/bin/hyperkube
-```
-### Master Nodes
-Start the kubelet:
-```
-sudo hyperkube kubelet --kubeconfig=/etc/kubernetes/kubeconfig \
-    --require-kubeconfig \
-    --cni-conf-dir=/etc/kubernetes/cni/net.d \
-    --network-plugin=cni \
-    --lock-file=/var/run/lock/kubelet.lock \
-    --exit-on-lock-contention \
-    --pod-manifest-path=/etc/kubernetes/manifests \
-    --allow-privileged \
-    --node-labels=master=true \
-    --minimum-container-ttl-duration=6m0s \
-    --cluster_dns=10.3.0.10 \
-    --cluster_domain=cluster.local \
-    --hostname-override=10.7.183.60
+Watch nodes spin up:
+
+```bash
+watch kubectl --insecure-skip-tls-verify --kubeconfig <(sed 's/kubernetes:443/192.168.77.10:443/' < assets/kubeconfig) get nodes
```

Comment:
per our discussion, using the -w flag might be better because it does not completely work in osx